common/src/main/java/io/netty/util/AsyncMapping.java

```diff
@@ -18,11 +18,11 @@
 import io.netty.util.concurrent.Future;
 import io.netty.util.concurrent.Promise;
 
-public interface AsyncMapping<IN, OUT> {
+public interface AsyncMapping<U, R> {
```

Comment:
For me `IN` and `OUT` are strange but `U` and `R` commonly used in JDK8+ functional interfaces and classes that build on top of it. E.g. [CompletableFuture](http://docs.oracle.com/javase/8/docs/api/java/util/concurrent/CompletableFuture.html#completedFuture-U-).

`U` as "user supplied value" and `R` as "return value".